charts/stable/common/README.md

```diff
@@ -224,6 +225,11 @@ All notable changes to this library Helm chart will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### [4.3.0]
+
+#### Added
+
+- Support for setting the runtimeClassName of the pods
```

Comment:
These changes will get overwritten since our README file gets automatically generated.

Please place these changes in the `README_CHANGELOG.md.gotmpl` file. Afterwards you can run `./hack/gen-helm-docs.sh stable common` again to update the chart's `README.md` file.

Reply:
Thanks for letting me know, it's corrected now.